src/Messages.js

```diff
@@ -1495,4 +1495,9 @@ export default defineMessages({
     description: 'No permissions label',
     defaultMessage: 'No permissions',
   },
+  copyToAll: {
```

Comment:
please run also `npm run translations` to generate the rest of translation files